assets/dev/js/editor/document/elements/commands/reset-settings.js

```diff
@@ -6,20 +6,24 @@ export class ResetSettings extends CommandHistory {
 	}
 
 	getHistory( args ) {
-		const { containers = [ args.container ] } = args;
+		const { containers = [ args.container ], options: { useHistory = true } = {} } = args;
```

Comment:
It should be a part of the `CommandHistory` not specific to this command/need.

Reply:
We'll discuss this.